packages/navigator/src/components/Navbar.tsx

```diff
@@ -126,6 +126,7 @@ const Navbar: React.FC<INavbarProps> = (props) => {
 
   return (
     <div
+      data-testid="nav-bar"
```

Comment:
```suggestion
      data-testid="navbar"
```

case는 파일명이랑 맞춰주는게 나을거같아요.

Reply:
data-testid는 빌드될때 사라지는지 궁금해요. Babel plugin 같은게 있나요?

Reply:
@tonyfromundefined 

> data-testid는 빌드될때 사라지는지 궁금해요. Babel plugin 같은게 있나요?

빌드 때 삭제할 수 있어요. [babel plugin](https://github.com/coderas/babel-plugin-jsx-remove-data-test-id) 도 있고요

당장은 필요없는 값이라 빌드 이후에 지울 수 있는데, 나중에 e2e 테스트 작성할 때도 사용할 수 있어서 "저는" 보통 prod 에 나갈 때도 남겨두는 편이에요.

data-testid 는 사용자 입장에서는 완전히 쓸모없는 값이라 data-testid 대신 aria-label 로 대체하는 방법도 있어요. 반대로 저는 aria-label 은 사용자에게 영향을 미치는 attribute 라서 사용을 꺼려요.

Reply:
아하 E2E때도 쓰이는군요 ㅋㅋ 좋아요~